dali/remote/daliservix.cpp

```diff
@@ -998,6 +1000,7 @@ class CRemoteFileServer
         registerCommand(RFCgetdir, &CRemoteFileServer::cmdGetDir);
         registerCommand(RFCgetcrc, &CRemoteFileServer::cmdGetCrc);
         registerCommand(RFCmove, &CRemoteFileServer::cmdMove);
+        registerCommand(RFCsetfileperms, &CRemoteFileServer::cmdSetFilePerms);
```

Comment:
There may be an issue here..
The regular impl. of dafilesrv (in sockfile.cpp) has other commands that have never been implemented in dafileservix (everything beyond version >= 1.5).
Which is 'okay', it rejects as cmd unknown.

1 option is not to implement setfileperms here either..

If it is implemented, need to be careful the enumerated value that follow RFCmove (i.e. RFCsetsize), doesn't incorrectly get seen as RFCsetfileperms.